tests/isBotInstalled.js

```diff
@@ -1,49 +1,69 @@
+/* eslint-disable no-unused-expressions */
 
-/**
- * Copyright (c) 2020, salesforce.com, inc.
- * All rights reserved.
- * Licensed under the BSD 3-Clause license.
- * For full license text, see LICENSE.txt file in the repo root or
- * https://opensource.org/licenses/BSD-3-Clause
- */
+// /**
```

Comment:
comment a comment ? :P

Reply:
:face_palm: haha will fix it, commented that file couple times during tests... probably left that behind. Thanks